ROADMAP.md

```diff
@@ -38,3 +38,12 @@ RPC server configuration setting SessionExpirationTime of `int` type has been
 replaced by SessionLifetime of `Duration` type.
 
 Removal of SessionExpirationTime is scheduled for 0.113.0 release.
+
+## Candidate registration via `registerCandidate` method of native NeoToken contract
+
+The original candidate registration command using the `registerCandidate` method
+of native NeoToken contract has been superseded by the GAS transfer approach.
+To use the candidate registration using the `registerCandidate` method, use the
+`--useRegisterCall` flag.
+
+Removal of `registerCandidate`–based registration is scheduled for 0.115.0 release.
```

Comment:
Removal of `registerCandidate`–based support of candidate registration will be done once `registerCandidate` method is officially deprecated and remove from the NeoToken manifest with the subsequent hardfork.